server/tilesource/base.py

```diff
@@ -287,58 +548,21 @@ def getRegion(self, width=None, height=None, **kwargs):
         :param height: maximum height in pixels.
         :param **kwargs: optional arguments.  Some options are encoding,
             jpegQuality, jpegSubsampling, top, left, right, bottom,
-            regionWidth, regionHeight, units ('pixels' or 'fraction').
+            regionWidth, regionHeight, units ('pixels' or 'fraction'),
+            magnification, mm_x, mm_y, exact.  See _tileIteratorInfo.
```

Comment:
to which all parameters does `units` apply to among - `width, height, regionWidth, regionHeight, top, left, right, bottom`?